src/sidebar/components/test/tag-editor-test.js

```diff
@@ -307,4 +307,34 @@ describe('TagEditor', function() {
       assert.isTrue(fakeOnEditTags.calledWith({ tags: ['tag2'] }));
     });
   });
+
+  describe('filter on text input based on user agent', () => {
+    let stub = sinon.stub(navigator, 'userAgent');
```

Comment:
This stub needs to happen in a `beforeEach` block which is executed before each test rather than in the top level of the `describe` block which is executed once at test setup time.